docs/source/build.rst

```diff
@@ -22,15 +22,12 @@ TorchAudio integrates PyTorch for numerical computation and third party librarie
 Customizing the build
 ---------------------
 
-TorchAudio's integration with third party libraries can be enabled/disabled via
+TorchAudio functionality can be enabled/disabled via
 environment variables.
 
 They can be enabled by passing ``1`` and disabled by ``0``.
 
-- ``BUILD_SOX``: Enable/disable I/O features based on libsox.
```

Comment:
There are many references to `sox`:
```sh
$ $ grep -r -I --exclude "*~" sox . | wc
     93     637   12170
```

Would it make sense to create a PR that removes all `sox` related stuff from the entire source tree at once?